app/src/main/java/com/soen490chrysalis/papilio/view/MainActivity.kt

```diff
@@ -24,8 +20,8 @@ class MainActivity : AppCompatActivity(), NavigationBarView.OnItemSelectedListen
         val view = binding.root
         setContentView(view)
 
-        //Setting the listener to detect when we press one of the button on the navigation bar
-        bottomNavigationView = binding.bottonnav
+        // Setting the listener to detect when we press one of the button on the navigation bar
+        bottomNavigationView = binding.bottomnav
```

Comment:
Did you run the code formatter on this file? I think it wasn't done

Reply:
I think this should be fixed now that I imported the configuration file you sent us